src/App.vue

```diff
@@ -394,6 +394,15 @@ body {
 
     &.active {
       right: 0;
+      display: initial;
+    }
+
+    &.top-level:not(.active) {
+      color: gold;
```

Comment:
Gold??? 😆

Reply:
Used that for testing (had some problems finding the right breakpoint)